pkg/tsdb/prometheus/time_series_query.go

```diff
@@ -202,7 +202,7 @@ func (s *Service) parseTimeSeriesQuery(queryContext *backend.QueryDataRequest, d
 	return qs, nil
 }
 
-func parseTimeSeriesResponse(value map[TimeSeriesQueryType]interface{}, query *PrometheusQuery) (data.Frames, error) {
+func parseTimeSeriesResponse(value map[TimeSeriesQueryType]interface{}, query *PrometheusQuery, fromAlerting bool) (data.Frames, error) {
```

Comment:
WDYT about renaming `fromAlerting` to something _like_ `fillNulls` and then using a pair of constants to set it? This would result in much more readable code.

Reply:
i was about to write the same comment.

`FromAlert` is too specific, while something like `FillNulls` can be reused and more straightforward.

Reply:
Great call! I'll update it.